src/screens/Accessibility/index.css

```diff
@@ -0,0 +1,11 @@
+input[type='range'] {
+  width: 100%;
+  max-width: 513px;
+}
+
+.Accessibility label,
+.Accessibility span,
+.Accessibility select,
+.Accessibility option {
+  font-size: 1.5rem;
```

Comment:
setting a bigger font size for the accessibility screen by default in case a user has troubles finding the zoom option with the original font size